src/components/CoinCard.tsx

```diff
@@ -4,11 +4,13 @@ import Image from "next/image";
 import { Button } from "./ui/button";
 import { useToast } from "@/components/ui/use-toast";
 import { Coin, Currency } from "@/lib/types";
+import { useState } from "react";
```

Comment:
Absolute imports should be at the top, followed by components and utils, and finally styles.